user/nvidia/template.py

```diff
@@ -0,0 +1,190 @@
+pkgname = "nvidia"
+pkgver = "560.35.03"
+pkgrel = 0
+archs = ["x86_64"]
+hostmakedepends = ["gcompat", "patchelf"]
+depends = [
+    # linked to
+    "libgcc-chimera",
+    "gcompat",
+    "openssl",
```

Comment:
I guess you can limit this dependency:
```suggestion
    "libcrypto3",
```